src/views/forms/FormNamespaceRegistrationTransaction/FormNamespaceRegistrationTransactionTs.ts

```diff
@@ -144,7 +144,7 @@ export class FormNamespaceRegistrationTransactionTs extends FormTransactionBase
         this.formItems.registrationType = this.registrationType || NamespaceRegistrationType.RootNamespace;
         this.formItems.newNamespaceName = this.namespaceId ? this.namespaceId.fullName : '';
         this.formItems.parentNamespaceName = this.parentNamespaceId ? this.parentNamespaceId.fullName : '';
-        this.formItems.duration = this.duration || 172800;
+        this.formItems.duration = this.duration || 86400;
```

Comment:
To access the connected node's network properties we can use `this.networkConfiguration`.

`this.formItems.duration = this.duration || this.networkConfiguration.minNamespaceDuration || 86400` ?